PROTOCOL.md

```diff
@@ -424,6 +424,37 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server.
+
+`|/team NUMBER|REQUEST ID`
+
+> Chooses which Pokemon to lead with.
+
+`|/choose move NUMBER|REQUEST ID` or `|/choose switch NUMBER|REQUEST ID`
+
+> Requests to use the selected move or switch to the selected Pokemon. The `REQUEST ID`
+> is used to identify which action the request was intended for to protect from cases
+> of trying to send multiple requests for the same action, which would otherwise effect
+> future decisions. The first decision of a match has a `REQUEST ID` of 1, and each
+> successful decision increments the counter by 1.
+> When a Pokemon is ko'd `/choose switch` is used to determine their replacement.
+
+`|/choose ACTION1 TARGET1,ACTION2 TARGET2|REQUEST ID`
+> For double and triple battles, decisions are sent for all team positions in the same
+> line separated by commas. Moves which specify a target list their chosen target as a
+> number wherein the opposing Pokemon are positive integers starting from `1` on the left
+> and incrementing up by one as they move to the right. Ally Pokemon targets are negative
+> integers starting from `-1` on the left and decrementing by one to the right. Moves
+> which don't need to specify a target send only the action.
+> If not all positions are making a decision, `pass` is sent in place of the action for
+> the position to be skipped.
```

Comment:
I looked at the documentation at the top of the battle messages, and how I described `POSITION` is very fitting for how to actually mention it.

>A Pokemon is always identified in the form `POSITION: NAME`. `POSITION` is
>the spot that the Pokemon is in: it consists of the `PLAYER` of the player
>(see `|player|`), followed by a letter indicating the given Pokemon's
>position, counting from `a`.

So basically you say that `NUMBER` is the position in the w/e, counting
upwards from `1`.

For this targeting part in Doubles, you'd say counting upwards from `1`
starting from the leftmost Pokemon? And counting downwards from `-1`
starting from the leftmost Pokemon for allies.

Actually, can you verify that `1` and `-1` start where you actually think
they start, just in case? Because in Triples, `p1a` and `p2c` face each
other-- your team counts starting from the left, while the opponent's
team would count starting from the right.